keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -1057,13 +1112,10 @@ and operation of this feature.
 Recall that end users cannot usually observe component logs or access metrics.
 -->
 
-- [ ] Events
-  - Event Reason: 
-- [ ] API .status
-  - Condition name: 
-  - Other field: 
-- [ ] Other (treat as last resort)
-  - Details:
+- [x] Events: All events for pulling images (failures and successes) also apply
+  to image volumes, because they re-use the same code paths. This makes it
+  consistent and straight forward for end users to verify what has been pulled
+  and if a Pod has been started successfully.
```

Comment:
For beta and beyond, I think it'd be good to help users see which image pulls were for volumes and which were for the code.

Reply:
I actually updated this section to use the metrics defined in the area above